lib/line/bot/client.rb

```diff
@@ -66,6 +66,10 @@ def blob_endpoint
                          end
       end
 
+      def liff_endpoint
+        @liff_endpoint ||= endpoint.sub(%r{/v2\z}, '')
```

Comment:
Could you define DEFAULT_LIFF_ENDPOINT?

example:
```
DEFAULT_LIFF_ENDPOINT = "https://api-data.line.me/liff/v1/"
```
(in LINE::Bot::API)

Reply:
Done!